.gitignore

```diff
@@ -110,3 +110,6 @@ dmypy.json
 
 # OSX files
 .DS_Store
+
+# Package lock?  May want to remove this
+package-lock.json
```

Comment:
Normally you want to keep the package-lock.json in version control for reproducible builds.